src/components/experiences/social-login/styled/icon.js

```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components';
+
+
+const StyledIcon = styled.svg`
```

Comment:
you can pass `styled(FacebookIcon)` instead `styled.svg`
since you create the styled of this way, you can't need pass `as={FacebookIcon}` in the index